docs/HOWTO/Software-Structure/Kitodo-Core/Role_management/new_role_management.md

```diff
@@ -1,63 +0,0 @@
-Role Management
```

Comment:
Why are you removing this documentation of current role management? 
@tim-boerner : could this file really be removed?

Reply:
The "current role management" page can be removed in my opinion, because now it is describing the the old approach. The "new role management" page is now describing the current approach, but it needs to be updated.

Reply:
Ok, so this file should be stay but must be updated for recent changes?

Reply:
I doubt that the "new role management" page is really useful for external developers in the current state. I would like to suggest to add a new page later.

Reply:
It is up to @felixlohmeier and @tim-boerner to decide. I think that this information get lost and nobody will reminder that this information was existing.

Reply:
In order to keep the docs clean for release we can remove it for now.